concepts/05 UI Components/Tabs/00 Getting Started with Tabs/15 Specify Selection Mode.md

```diff
@@ -0,0 +1,66 @@
+Users can select Tabs items in two different modes: `'single'` (default) or `'multiple'`. You can use the [selectionMode](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectionMode) property to change the mode. To preselect or to select an item programmatically, pass its index in the data source array to the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedIndex) property. As an alternative, you can use the [selectedItem](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedItem) (for `'single'` selection mode) or [selectedItems](/Documentation/ApiReference/UI_Components/dxTabs/Configuration/#selectedItems) (for `'multiple'` selection mode) properties.
```

Comment:
`Users can select Tabs items in two different modes:`

Is "Tabs" the name of the component? If so, can we say "Tabs component items"? Or do you mean "Tab items"?

Reply:
I'll change it to "Tabs component items"